project/LinkerdBuild.scala

```diff
@@ -369,6 +369,14 @@ object LinkerdBuild extends Base {
 
     val examples = projectDir("namerd/examples")
       .withExamples(Namerd.all, exampleConfigs)
+
+    // TODO: remove or consolidate dependencies
+    val dcosBootstrap = projectDir("namerd/dcos-bootstrap")
+      .dependsOn(core, configCore, Router.http,
+        Iface.controlHttp, Iface.interpreterThrift,
```

Comment:
Why don't we just go the other way and pull dcosBootstrap into the Bundle configuration (so it'll have access to all of the bundled plugins)?